autoscaler/controllers/gateway/config/elasticsearch.go

```diff
@@ -36,7 +45,7 @@ func (e *Elasticsearch) ModifyConfig(dest *odigosv1.Destination, currentConfig *
 		}
 
 		currentConfig.Exporters[esTraceExporterName] = commonconf.GenericMap{
-			"endpoints":    []string{fmt.Sprintf("%s:9200", url)},
+			"endpoints":    []string{parsedURL},
```

Comment:
Since the current odigos version always appends the port, I assume that people configure the endpoint with no port (something like `https://elastic.example.com`). Thus, if we now stop appending the port they might break.

What do you think about checking if the port exists in the `SanitizeURL` function -
- It the user supplied a port in the URL use it.
- If there is no port in the URL, we can fallback to using `9200` which will keep existing users configuration working.

You can consider using the [urlHostContainsPort](https://github.com/keyval-dev/odigos/blob/206ef74eccff945a582571186365d2410d76347d/autoscaler/controllers/gateway/config/utils.go#L73) function to do that.

Reply:
this is done ✅